src/Apps/Legion/Pages/FlowPickerPage/FlowPickerPage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { getDebugLogger } from '../../../../Models/Services/Utils';
+import { IFlowPickerPageProps } from './FlowPickerPage.types';
+import { Stack } from '@fluentui/react';
+import FlowPicker from '../../Components/FlowPicker/FlowPicker';
+
+const debugLogging = false;
+const logDebugConsole = getDebugLogger('FlowPickerPage', debugLogging);
+
+const FlowPickerPage: React.FC<IFlowPickerPageProps> = () => {
+    // contexts
+
+    // state
+
+    // hooks
+
+    // callbacks
+
+    // side effects
+
+    // styles
+
+    logDebugConsole('debug', 'Render');
+
+    return (
+        <Stack tokens={{ childrenGap: 8 }}>
+            <h2>Pick a flow</h2>
+            <FlowPicker />
+        </Stack>
+    );
+};
+
+export default FlowPickerPage;
```

Comment:
I don't think FlowPicker should be a separate "page", should be a "component" instead.

Reply:
Yeah. right now it's pretty thin but we will eventually have more to show on the pages that we don't have to incorporate into the components. I'd like to keep the components pretty focused on their purpose rather than mega components that do everything on a page. This is thin for now but I expect it will have some more things in the future. It also just makes it cleaner to have components that don't need props for navigation so the pages can take care of initializing all the components on that particular screen.